manifests/cli-manager-operator.clusterserviceversion.yaml

```diff
@@ -84,13 +84,25 @@ spec:
           - climanagers
           - climanagers/status
           verbs:
-          - '*'
+          - create
```

Comment:
I think, we should add deletecollection for these resource because we are the owner.